webpack.common.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */ 
```

Comment:
you need to check all errors associated with javascript so that you do not disable it because this might cause problems when working on a large problem.